cli/create.py

```diff
@@ -29,12 +29,12 @@ def handle_rsa_keys(total: int) -> Tuple[RsaKey, RsaKey, List[str]]:
     Generates RSA keypair for communicating with other horcruxes
     and retrieves all the other horcruxes RSA public keys.
     """
-    print("Generating RSA key for communicating with other horcruxes...")
+    click.echo("Generating RSA key for communicating with other horcruxes...")
     my_rsa_private_key = RSA.generate(4096)
     my_rsa_public_key = my_rsa_private_key.publickey()
 
-    print(f'\n\n{my_rsa_public_key.export_key("OpenSSH").decode("ascii")}')
-    print("\n\nShare the RSA public key above with all other horcruxes")
+    click.echo(f'\n\n{my_rsa_public_key.export_key("OpenSSH").decode("ascii")}')
```

Comment:
I think we can use a different color here to emphasize the public key:
```suggestion
    click.secho(f'\n\n{my_rsa_public_key.export_key("OpenSSH").decode("ascii")}', fg="green")
```